include/mpm_explicit_usf.tcc

```diff
@@ -124,6 +124,34 @@ bool mpm::MPMExplicitUSF<Tdim>::solve() {
         std::bind(&mpm::ParticleBase<Tdim>::compute_strain,
                   std::placeholders::_1, phase, dt_));
 
+    // Assign pressure to nodes
+    mesh_->iterate_over_particles(
+        std::bind(&mpm::ParticleBase<Tdim>::map_pressure_to_nodes,
+                  std::placeholders::_1, phase));
+
+#ifdef USE_MPI
+    // Run if there is more than a single MPI task
+    if (mpi_size > 1) {
+      // MPI all reduce nodal pressure
+      mesh_->allreduce_nodal_scalar_property(
+          std::bind(&mpm::NodeBase<Tdim>::pressure, std::placeholders::_1,
+                    phase),
+          std::bind(&mpm::NodeBase<Tdim>::update_pressure,
+                    std::placeholders::_1, false, phase, std::placeholders::_2,
+                    std::placeholders::_2));
+    }
```

Comment:
I can't use `std::placeholders::_3` for some reason. Note that `update_pressure` has 4 input arguments: `update boolean`, `phase`, `mass` and `pressure`.